src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -36,6 +36,12 @@
 
 logger = logging.get_logger(__name__)
 
+if torch.backends.cuda.matmul.allow_fp16_reduced_precision_reduction is False:
```

Comment:
This surely doesn't work for past PyTorch versions no? Also is this really necessary?

Reply:
Actually this warning message is completely optional

Reply:
Patrick's point is that the test will raise an error for old versions of PyTorch where `torch.backends.cuda.matmul.allow_fp16_reduced_precision_reduction` does not exist.

Reply:
Fixed in ed9682dcdd7366e2a95a3e021c37a2434fda6a24

Reply:
I think that I will change the tests to fp32 to avoid these changes/errors as suggested by Patrick